accesscontroltool-package/src/main/jcr_root/apps/netcentric/actool/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-ncActoolSystemUser.config

```diff
@@ -1 +1 @@
-user.mapping=["biz.netcentric.cq.tools.accesscontroltool.bundle\=actool-service","biz.netcentric.cq.tools.accesscontroltool.startuphook.bundle\=actool-service"]
\ No newline at end of file
+user.mapping=["biz.netcentric.cq.tools.accesscontroltool.bundle\=[actool-service]","biz.netcentric.cq.tools.accesscontroltool.startuphook.bundle\=[actool-service]"]
```

Comment:
IIRC I had problems using this syntax (I think it worked locally but not in the cloud service), have you tested in cloud service if it works like this?

Reply:
For cloud use it since a long time already: https://github.com/Netcentric/accesscontroltool/blob/develop/accesscontroltool-package/src/main/jcr_root-cloud/apps/netcentric/actool/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-ncActoolSystemUser.config